mosquitto/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: mosquitto_broker_1
+      APP_PORT: 9001
+    container_name: mosquitto_app_proxy_1
+  broker:
+    image: eclipse-mosquitto:2.0.20@sha256:8b396cec28cd5e8e1a3aba1d9abdbddd42c454c80f703e77c1bec56e152fa54e
+    restart: on-failure
+    ports:
+      - '1883:1883'
+      - '9001:9001'
```

Comment:
Port 1883 is free, but 9001 is used by Penpot. You could change to `9991:9001` and update the instructions accordingly.